### PR TITLE
Coq num analysis.1.0.0

### PR DESCRIPTION
**Pull request by mmayero**
Package for Numerical Analysis in Coq: Lax Milgram th, Lebesgue integration, Tonell th, Bochner th.

### CODE REVIEW

File: released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Numerical Analysis in Coq"
+name: "coq-num-analysis"
+version: "1.0.0"
```
Comment (palmskog) [inline released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam:4]: The package name and version is already given by the file path (`coq-num-analysis.1.0.0/opam`), so avoid them inside the file.
```suggestion
```

File: released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+maintainer: "MILC project <milc@inria.fr>"
+authors: [ "Sylvie Boldo" "François Clément" "Vincent Martin" "Micaela Mayero" "Florian Faissole" "Houda Mouhcine" "Louise Leclerc" "Stéphane Aubry" ]
+homepage: "https://lipn.univ-paris13.fr/coq-num-analysis/"
+dev-repo: "git+https://lipn.univ-paris13.fr/coq-num-analysis/"
+bug-reports: "https://lipn.univ-paris13.fr/coq-num-analysis/issues"
+license: "LGPL-3.0-or-later"
+tags: [ "keyword:real analysis" "keyword:Lax Milgram" "keyword:Lebesgue" "keyword:Tonelli" "keyword:Bochner"]
+build: [make]
```
Comment (palmskog) [inline released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam:12]: Usually, we want to allow file parallelism to allow the package to build faster on multicore machines. Is there are a reason not to?
```suggestion
build: [make "-j%{jobs}%"]
```

File: released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+install: [make "install"]
+remove: [make "uninstall"]
```
Comment (palmskog) [inline released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam:14]: If this is a regular Coq library, it's best to let opam take care of tracking files for installation and removal.
```suggestion
```

**Review by palmskog — CHANGES_REQUESTED** [review]
There is a problem with the file URL:
```
[ERROR] Failed to get sources of coq-num-analysis.1.0.0: Bad checksum
```
I recommend making a [Git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) in the repo, which will automatically give a tarball, e.g., a previous tag has this URL: https://depot.lipn.univ-paris13.fr/mayero/coq-num-analysis/-/archive/Bochner.1.0/coq-num-analysis-Bochner.1.0.tar.gz

File: released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+depends: [
+  "ocaml" { >= "4.08.1" }
```
Comment (silene) [inline released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam:16]: Does the library come with a plugin? If not, I suggest to remove the dependency on OCaml:
```suggestion
```

File: released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+synopsis: "Numerical Analysis in Coq"
+name: "coq-num-analysis"
+version: "1.0.0"
+maintainer: "MILC project <milc@inria.fr>"
+authors: [ "Sylvie Boldo" "François Clément" "Vincent Martin" "Micaela Mayero" "Florian Faissole" "Houda Mouhcine" "Louise Leclerc" "Stéphane Aubry" ]
+homepage: "https://lipn.univ-paris13.fr/coq-num-analysis/"
+dev-repo: "git+https://lipn.univ-paris13.fr/coq-num-analysis/"
+bug-reports: "https://lipn.univ-paris13.fr/coq-num-analysis/issues"
+license: "LGPL-3.0-or-later"
+tags: [ "keyword:real analysis" "keyword:Lax Milgram" "keyword:Lebesgue" "keyword:Tonelli" "keyword:Bochner"]
```
Comment (silene) [inline released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam:11]: I suggest adding a few more tags for the purpose of accurate metadatas:
- `"category:..."` for proper indexing in  https://coq.inria.fr/opam/www/
- `"date:2022-08-..."` for the release date of the library (not of the Opam package)
- `"logpath:..."` for the Coq logical path where the library gets installed

**Comment by palmskog** [comment]
@mmayero the key issue here is that we need a tag in your repository to get stable archive for the package. Could you create a tag called something like `1.0` in your Git repository? If so, I can take care of modifying the package here afterwards.

**Comment by mmayero** [comment]
@palmskog I have pushed the tag "1.0" in our GIT repository. It compiles with Coq 8.15. I have not modified category and tag in opam file but may be I can do it later(?) Or else it should be:

category: "Real Calculus and Topology"

and modify the

tags: [ "keyword:real analysis" "keyword:Lax Milgram Theorem" "keyword:Lebesgue Integration" "keyword:Tonelli Theorem" "keyword:Bochner Integration"]

**Comment by palmskog** [comment]
@mmayero great, now we have a tarball. Maybe easiest if I take care of the package definition updates, I'll incorporate the category and keywords.

**Review by palmskog — APPROVED** [review]

**Pull request merged**